test/rpc_test.py

```diff
@@ -14,6 +14,7 @@
 from common_utils import load_tests
 import dist_utils
 from dist_utils import dist_init
+from torch.distributed.rpc.api import _use_rpc_pickler
```

Comment:
nit: can we put this in `torch.distributed.rpc` and not `torch.distributed.rpc.api`?